src/middlewared/middlewared/plugins/iscsi_/extents.py

```diff
@@ -401,7 +408,7 @@ async def save(self, data, schema_name, verrors):
 
             # create the extent
             if not os.path.exists(path):
-                await run(['truncate', '-s', str(data['filesize']), path])
+                subprocess.run(['truncate', '-s', str(data['filesize']), path])
 
```

Comment:
Looking at implementation of truncate(1) in coreutils, I wonder if it would be better to implement in pure python, e.g. os.open(); os.ftruncate().

Reply:
Thx @anodos325 -- I'll be more than happy to do that, but probably under the aegis of a separate PR.